aweadmin.py

```diff
@@ -211,6 +258,8 @@ def main():
       showjobs=True
     elif arg == '-n':
```

Comment:
Totally agree. It kind of bugged me to not do it while I was in there, but I didn't have time to verify that there wasn't some historical reason to leave it as it was.